test-project-setup.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+setup () {
+  # TEAM FOLDER must be in set in local environment
+  if [[ ! ${TEAM_FOLDER} ]]
+  then echo "TEAM_FOLDER missing"; exit
+  fi
+
+  export PARENT_PROJECT=$(gcloud config get-value project)
+  # PARENT PROJECT required for getting billing account
+  if [[ ! ${PARENT_PROJECT} ]]
+  then echo "PARENT_PROJECT missing"; exit
+  fi
+
+  export BILLING_ACCOUNT=$(gcloud beta billing projects describe ${PARENT_PROJECT} --format="value(billingAccountName)" || sed -e 's/.*\///g')
+  # BILLING ACCOUNT required for API enablement
+  if [[ ! ${BILLING_ACCOUNT} ]]
+  then echo "BILLING_ACCOUNT missing"; exit
+  fi
+
+  export TESTING_PROJECT=$(printf "cloud-run-samples-test-%06d" $((RANDOM%999999)))
```

Comment:
In the case of golang-samples it's often been useful to do things like this:

```
for i in {1..7}; do gcloud --project cloud-run-samples-test-$i tasks queue create my-queue; done;
```

This implies we may want to have sequential numbers.

Reply:
Now uses a suffix set in an environment variable.